src/verify.py

```diff
@@ -0,0 +1,23 @@
+#Write a software that verifies if a number is present in a pre-defined array.
+#Output example:
+#Insert number 3
+#The number 3 is [not] present in the array.
+
+def main():
+    N = [3, 4, 5, 1, 2, 3, 4, 9, 13, 0]
+    numero_da_trovare = int(input("Insert number: "))
+    found = False
+
+    for num in N:
+        if num == numero_da_trovare:
+            found = True
+            break

```

Comment:
se "estrai" questa parte di codice in una funzione, potresti evitare di utilizzare la variabile found